recipe/0068-newspaper/newspaper_issue_2-manifest.json

```diff
@@ -0,0 +1,176 @@
+      "type": "Image",
+      "format": "image/jpeg"
+    }
+  ],
+  "navDate": "1925-03-13T00:00:00Z",
+  "rights": "http://creativecommons.org/publicdomain/mark/1.0/",
+  "requiredStatement": {
+    "label": {
+      "en": [
+        "Attribution"
+      ]
+    },
+    "value": {
+      "en": [
+        "Berliner Tageblatt - https://www.europeana.eu/portal/record/9200355/BibliographicResource_3000096302605.html. Staatsbibliothek zu Berlin - Preußischer Kulturbesitz. Public Domain Mark - http://creativecommons.org/publicdomain/mark/1.0/"
+      ]
+    }
+  },
+  "logo": [
+    {
+      "id": "https://style.europeana.eu/images/europeana-logo-default.png"
+    }
+  ],
+  "seeAlso": [
+    {
+      "id": "https://www.europeana.eu/api/v2/record/9200355/BibliographicResource_3000096302516.json-ld",
+      "format": "application/ld+json",
+      "profile": "http://www.europeana.eu/schemas/edm/"
+    }
+  ],
+  "start": {
+    "id": "https://iiif.europeana.eu/presentation/9200355/BibliographicResource_3000096302516/canvas/p1",
+    "type": "Canvas"
+  },
+  "items": [
+    {
+      "id": "https://iiif.europeana.eu/presentation/9200355/BibliographicResource_3000096302516/canvas/p1",
+      "type": "Canvas",
+      "label": {
+        "none": [
+          "p. 1"
+        ]
+      },
+      "height": 5000,
+      "width": 3602,
+      "items": [
+        {
+          "id": "https://iiif.europeana.eu/presentation/9200355/BibliographicResource_3000096302516/annotation_page_painting/ap1",
+          "type": "AnnotationPage",
+          "items": [
+            {
+              "id": "https://iiif.europeana.eu/presentation/9200355/BibliographicResource_3000096302516/annotation/p1",
+              "type": "Annotation",
+              "motivation": "painting",
+              "body": {
+                "id": "https://iiif.europeana.eu/image/3UU6R3RRZZGU2VNISCQX7N474GR7X4VMGYBTIWV2SNCBRGSR2WAA/presentation_images/ea1ba210-ffd3-11e5-b68d-fa163e60dd72/node-2/image/SBB/Berliner_Tageblatt/1925/03/13/0/F_SBB_00001_19250313_054_123_0_001/full/full/0/default.jpg",
+                "type": "Image",
+                "format": "image/jpeg",
+                "service": [
+                  {
+                    "id": "https://iiif.europeana.eu/image/3UU6R3RRZZGU2VNISCQX7N474GR7X4VMGYBTIWV2SNCBRGSR2WAA/presentation_images/ea1ba210-ffd3-11e5-b68d-fa163e60dd72/node-2/image/SBB/Berliner_Tageblatt/1925/03/13/0/F_SBB_00001_19250313_054_123_0_001",
+                    "type": "ImageService3",
+                    "profile": "level2"
+                  }
+                ]
+              },
+              "target": "https://iiif.europeana.eu/presentation/9200355/BibliographicResource_3000096302516/canvas/p1"
+            }
+          ]
+        }
+      ],
+      "annotations": [
+        {
+          "id": "https://iiif.europeana.eu/presentation/9200355/BibliographicResource_3000096302516/annopage/1",
```

Comment:
This looks to be a version 2.0 annotation list rather than 3.0 annotations. Could we convert them to 3.0 and host them locally?